gateway/gateway_test.go

```diff
@@ -226,7 +226,7 @@ var _ = Describe("Gateway Enterprise", func() {
 		It("should accept events from normal users", func() {
 			allowedUserEventData := fmt.Sprintf("{\"batch\":[{\"userId\": \"%s\"}]}", NormalUserID)
 
-			mockCall := c.mockJobsDB.EXPECT().StoreWithRetryEach(gomock.Any()).DoAndReturn(jobsToEmptyErrors).Times(1)
+			mockCall := c.mockJobsDB.EXPECT().AppendWithRetryEach(gomock.Any()).DoAndReturn(jobsToEmptyErrors).Times(1)
```

Comment:
be aware here we do not test the arguments of AppendWithRetryEach, I assume we don't have to right?